### ISSUE TITLE
[Documentation] [Study] Alpha build of documentation

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#353]
# Context

Some features can be alpha-tested, like for example components and modules.
These features must be documented, i.e. have their Swift file updated and mauybe the DocC catalogs.
The documentation process is not 100%-automatized: we need to use Xcode manualy to build the doccarchive, then run the script to build the HTML doc.
It can be interesting to have a preview of the HTML documentation for the PO for example. We can imagine to use Netlify to load HTML content in a *docs* folder in the feature branch. This *docs* folder will be populated by the documentation script but without updating the *GitHub Pages* branch.

For example, using the command:

```shell
uploadWebDoc.sh --libversion=\#XXX 
# Do not add --usegit option so as to not update the GitHub branch
# Replace libversion value by for example the issue number
```

# Definition of Ready

- [ ] Check of works Netlify
- [ ] Check if possible to provide documentation on a feature branch in a dedicated folder
- [ ] Chekc if the doc script can be updated to as to copy the THML assets from temporary folder to another one

**Comment by florentmaitre** [comment]
Yes, I can help configuring Netlify and explain how we deploy our documentation there 👍
